known_issues: Add issue NCSDK-8321 with the shell_bt_nus sample (#4037)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0

NCSDK-8321: NUS shell transport sample doesn't display the initial shell prompt *uart:~$* on the remote terminal.
  Also few logs with sending errors are displayed on the terminal connected directly to the DK.

>   This issue is caused by the shell enabling before turning on the notification for the NUS service by the remote peer.

Reviewer: ```suggestion
  This issue is caused by the shell being enabled before turning on the notifications for the NUS service by the remote peer.
```

Author: Done

Reviewer: One more minor thing: We should avoid using contractions (isn't, doesn't etc) in the docs. This is not very bad, but maybe we should have does not instead of doesn't.

Author: Ok. I changed to the full form

>   **Workaround:** Enabling the shell should be done after turning on the NUS notification or should be blocked until the notification is turned on.

Reviewer: ```suggestion
  **Workaround:** Enable the shell after turning on the NUS notifications or block it until turning on the notifications.
```

Author: Done
